README.md

```diff
@@ -62,7 +45,57 @@ To access logs directory, go to Help -> Show log in IDE.
   <kbd>Settings/Preferences</kbd> > <kbd>Plugins</kbd> > <kbd>⚙️</kbd> > <kbd>Install plugin from disk...</kbd>
 
 
+### Dependencies
+
+* [Python](https://www.python.org/downloads/) >= 3.7 or [Pipenv](https://docs.pipenv.org/) or [Docker](https://www.docker.com/products/docker-desktop) daemon running
+
+The Checkov extension will invoke the latest version of ```Checkov```.
+
+### Configuration
+
+* Sign up to a Bridgecrew Community account [here](http://bridgecrew.cloud/). If you already have an account, sign in and go to the next step.
+
+* From [Integrations](https://www.bridgecrew.cloud/integrations/api-token), select **API Token** and copy the API key.
+* In Jetbrains, enter your API Token in the Checkov plugin settings page under tools.  
+* Using a custom CA certificate is possible. If needed, set the path to the certificate file in the Checkov plugin settings page.
+
+### Usage
+
+* Open a file you wish to scan with checkov in IntelliJ.
+* Checkov will run automatically everytime an IaC is opened or saved.
+* Scan results should now appear in the checkov tool window in the bottom of your IDE.
+* Scan results will appear on the left side as a tree of File Names -> Resources -> Violated checks.
+* Click a check to see its details. Details including  violating policy and a link to step-by-step fix guidelines.
+* In most cases, the Details will include a fix option. This will either add, remove or replace an unwanted configuration, based on the Checkov fix dictionaries.
+* You can skip checks by adding an inline skip annotation ```checkov:skip=<check_id>:<suppression_comment>```. For more details see the [docs](https://github.com/bridgecrewio/checkov/blob/master/docs/2.Concepts/Suppressions.md).
+
+### Troubleshooting logs
+
+To access checkov-intellij logs directory, go to `Help` and select `Show Log in Finder`.
```

Comment:
```suggestion
To access checkov-intellij logs directory, go to `Help` and select `Show Log in Finder` or `Show Log in Explorer`.
```

Reply:
Perfect, the second is for windows right? so it makes sense if I add (for Mac) and (for Windows) after each @orhovy

Reply:
yes